.github/workflows/phpunit.yaml

```diff
@@ -0,0 +1,56 @@
+name: PHPUnit-unit-test
+
+# Run this workflow every time a new commit pushed to your repository
+on: push
+
+jobs:
+  # Set the job key. The key is displayed as the job name
+  # when a job name is not provided
+  phpunit:
+    # Name the Job
+    name: unit test API code
+    # Set the type of machine to run on
+    runs-on: ubuntu-latest
+    env:
+      DEFAULT_BRANCH: master
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+    # Checks out a copy of the repository on the ubuntu-latest machine
+    - name: Checkout code
+      uses: actions/checkout@v2
+      
+      # Run the docker environment
+    - name: Build the docker-compose stack
+      run: docker-compose up -d
+    
+    # Wait to ensure containers are up
+    - name: pause to allow conainers to spin up
+      uses: jakejarvis/wait-action@master
+      with:
+        time: '30s'
+        
+    # check docker container status
+    - name: Check running containers
+      run: docker ps
+      
+    # prepare database
+    #- name: create database using doctrine ORM
```

Comment:
Unnecessary comment ?

Reply:
removed, was used to 'create' database before migrations